### DISCUSSION TITLE
Dia_NN stopped

### DISCUSSION THREAD

**Discussion by rajeshksoni11** [vdemichev/DiaNN#114] [General]
Hi,
I am seeing this message, any suggestions to fix this errors.

Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
Only peaks with correlation sum exceeding 1 will be considered
Peaks with correlation sum below 1 from maximum will not be considered
DIA-NN exited
A single score will be used until RT alignment to save memory; this can potentially lead to slower search
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
DIA-NN-plotter.exe "C:\DIA-NN\1.7.16\report.stats.tsv" "C:\DIA-NN\1.7.16\report.tsv" "C:\DIA-NN\1.7.16\report.pdf"
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
Mass accuracy will be fixed to 1e-05 (MS2) and 1e-05 (MS1)

1 files will be processed
[0:00] Loading spectral library D:\Data\CHU-DIA_NN\EasyPQP_lib\psm.tsv
WARNING: cannot find column  PrecursorMz "PrecursorMz" Q1 
ERROR: C:\diann\src\diann.cpp: 13621: Cannot load spectral library

PDF report will be generated in the background

diann.exe --f "D:\Data\CHU-DIA_NN\Chu_diaPASEF_8R3_Slot2-9_1_949.d
" --lib "D:\Data\CHU-DIA_NN\EasyPQP_lib\psm.tsv" --threads 8 --verbose 1 --out "C:\DIA-NN\1.7.16\report.tsv" --qvalue 0.01 --matrices  --min-corr 1.0 --corr-diff 1.0 --time-corr-only --fasta "C:\Users\Admin\Desktop\Database\uniprot-proteome_UP000000589+reviewed_yes.Fasta" --met-excision --cut K*,R* --mass-acc 10 --mass-acc-ms1 10 --smart-profiling --peak-center --no-ifs-removal 
DIA-NN 1.7.16 (Data-Independent Acquisition by Neural Networks)
Compiled on Mar 27 2021 21:34:06
Current date and time: Mon May 31 09:51:31 2021
CPU: GenuineIntel Intel(R) Xeon(R) W-3225 CPU @ 3.70GHz
SIMD instructions: AVX AVX2 AVX512CD AVX512F FMA SSE4.1 SSE4.2 
DIA-NN exited
Logical CPU cores: 16
Thread number set to 8
Output will be filtered at 0.01 FDR
DIA-NN-plotter.exe "C:\DIA-NN\1.7.16\report.stats.tsv" "C:\DIA-NN\1.7.16\report.tsv" "C:\DIA-NN\1.7.16\report.pdf"
Precursor/protein x samples expression level matrices will be saved along with the main report
Only peaks with correlation sum exceeding 1 will be considered
Peaks with correlation sum below 1 from maximum will not be considered
PDF report will be generated in the background

A single score will be used until RT alignment to save memory; this can potentially lead to slower search
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
Mass accuracy will be fixed to 1e-05 (MS2) and 1e-05 (MS1)

1 files will be processed
[0:00] Loading spectral library D:\Data\CHU-DIA_NN\EasyPQP_lib\psm.tsv
WARNING: cannot find column  PrecursorMz "PrecursorMz" Q1 
ERROR: C:\diann\src\diann.cpp: 13621: Cannot load spectral library

**Comment by vdemichev** [comment]
Hi, sorry for late reply, it seems I am not getting proper notifications for posts in 'Discussions'. Basically, what happens is that DIA-NN does not support a particular spectral library format. Please contact FragPipe support on how to generate spectral libraries compatible with DIA-NN.
